.github/workflows/CI.yml

```diff
@@ -93,7 +92,37 @@ jobs:
       with:
         name: binary_artifacts
         path: build
+        
+  analyze:
+    name: CodeQL Analyze
+    needs: build
+    runs-on: ubuntu-latest
 
+    strategy:
+      fail-fast: false
+      matrix:
+        language: [ 'go' ]
```

Comment:
i guess we don't need matrix if this workflow is just for aoc, which is a golang project.